site/documentation/content/api/Metrics.md

```diff
@@ -74,6 +74,7 @@ Metrics provided by the protocol adapters are:
 | *hono.commands.payload*            | DistributionSummary | *host*, *component-type*, *component-name*, *tenant*, *type*, *status*, *direction*          | The number of bytes conveyed in the payload of a command message. |
 | *hono.connections.authenticated*   | Gauge               | *host*, *component-type*, *component-name*, *tenant*                                         | Current number of connected, authenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.unauthenticated* | Gauge               | *host*, *component-type*, *component-name*                                                   | Current number of connected, unauthenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
+| *hono.connections.authenticated.duration* | Timer        | *host*, *component-type*, *component-name*, *tenant*                                         | The length of time for which the authenticated devices stay connected to Hono. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
```

Comment:
how about this for description:
The overall amount of time that authenticated devices have been connected to protocol adapters. **NB** ...

Reply:
It sounds even better. I have changed it and pushed the commit.

Reply:
Are you also adding unit tests to this PR?

Reply:
yes, I am working on it now.

Reply:
I have added tests. Would you mind taking a look?